client/rest-high-level/src/test/java/org/elasticsearch/client/SearchIT.java

```diff
@@ -1233,4 +1235,71 @@ private static void assertSearchHeader(SearchResponse searchResponse) {
         assertEquals(0, searchResponse.getShardFailures().length);
         assertEquals(SearchResponse.Clusters.EMPTY, searchResponse.getClusters());
     }
+
+    //Count API IT tests start
```

Comment:
you dont need these, its a big ole file but we dont do it with other sections

Reply:
ok, removed